updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml

```
@@ -0,0 +1,164 @@
+name: Open a PR that prepares the policy for release
+pipelineid: "{{ .pipelineid }}"
```

```suggestion
pipelineid: "{{ .pipelineid }}"
version: v0.97.0
```

This ensures that Updatecli reports a warning as the manifest only work with Updatecli v0.97.0 or greater
As this manifest require @viccuad pull request [feat: Support key title for githubrelease source, condition](https://github.com/updatecli/updatecli/pull/4528)